evap/static/scss/components/_confirmation-modal.scss

```diff
@@ -0,0 +1,87 @@
+confirmation-modal:not(:defined) > :not([slot="submit-group"]) {
+  // Without this, the elements that are slotted into the dialog element show up as if they are just normal child
+  // elements and disappear once the custom element registration is done. To avoid a short flicker of these elements,
+  // we hide them until the constructor of the custom element has run.
+  display: none;
+}
+
+dialog.evap-modal-dialog {
+  z-index: $zindex-modal;
+  max-width: 80ch;
+  border: 1px solid $dark-gray;
+  border-radius: 0.5rem;
+  padding: 0;
+
+  // These attributes are inherited from parent elements, although it usually does not make any sense, because the
+  // dialog is not displayed inside of the parent element. This list might grow over time.
+  font-weight: initial;
+  line-height: initial;
+  cursor: initial;
+  text-align: initial;
+
+  &[open] {
+    animation: modal-enter 300ms forwards;
+  }
+
+  &[closing] {
```

Comment:
I actually did this on purpose, because pressing Escape is not handled by us at all and is just built-in functionality, so I did not want to interfere with it. I also prefer the current behavior, but if the group consensus is different, I can also change it

Reply:
On current main, escape and pressing X behave identically, and I think that is a bit nicer, but I don't care too much. We can merge as is if @janno42 is fine with it.

Reply:
Fine with it :)